src/Appwrite/Utopia/Response/Model/File.php

```diff
@@ -38,14 +38,14 @@ public function __construct()
                 'type' => self::TYPE_STRING,
                 'description' => 'File read permissions.',
                 'default' => [],
-                'example' => 'role:all',
+                'example' => ['role:all'],
```

Comment:
<img width="801" alt="Screen Shot 2022-08-11 at 1 58 29 PM" src="https://user-images.githubusercontent.com/29069505/184206243-2bde8ccd-f151-4637-8847-41419bdd9a8d.png">

Reply:
Although that mime type looks super weird.